configuration/debian/tedge/preinst

```diff
@@ -3,13 +3,13 @@ set -e
 
 ### Create groups
 if ! getent group tedge >/dev/null; then
-    addgroup --quiet --system tedge
+    groupadd -r tedge
 fi
 
 ### Create users
 # Create user tedge with no additional info(--gecos "") no home(--no-create-home), no login(--shell) and in group tedge(--ingroup)
 if ! getent passwd tedge >/dev/null; then
-    adduser --quiet --system --gecos "" --no-create-home --disabled-login --shell /sbin/nologin --ingroup tedge tedge
+    useradd -r -M -s /sbin/nologin -g tedge tedge
```

Comment:
I was looking into [Linux man page](https://linux.die.net/man/8/useradd) for `useradd` and there was no information about `--no-create-home` but it is working according to my tests so that can be changed. `--system` flag just like the previous comment. We can also change `-g` to `--gid` and `-s` to `--shell` if that is more explicit.